cmake_modules/FindSDE.cmake

```diff
@@ -0,0 +1,37 @@
+# - Find the SDE library (part of PAPI)
+# This module finds an installed  lirary that implements the
+# Softward Defined Events interface, part of the PAPI Project 
+# (see http://icl.cs.utk.edu/papi/).
+#
+# This module defines the PAPI::SDE target.
+##########
+
+find_path(SDE_INCLUDE_DIR sde_lib.h
+          PATHS ${PAPI_ROOT}/include ENV PAPI_INCLUDE_DIR
```

Comment:
I guess `PAPI_ROOT` (or `PAPI_INCLUDE_DIR` and `PAPI_LIBRARY_DIR`) needs to be passed to CMAKE, right? Because I found the module of `papi/5.7.0` on saturn does not provide `PAPI_INCLUDE_DIR` and `PAPI_LIBRARY_DIR`.

Reply:
PAPI_INCLUDE_DIR would be the output variable from findPAPI, I'd say we should only use PAPI_INCLUDE_DIR and not PAPI_ROOT on this secondary stage, but... 

given that SDE is a subcomponent of PAPI, this should be done during findPAPI so that we can use the COMPONENTS keyword during find_package(PAPI)

```
find_package(<PackageName> [version] [EXACT] [QUIET] [MODULE]
             [REQUIRED] [[COMPONENTS] [components...]]
             [OPTIONAL_COMPONENTS components...]
```
see https://cmake.org/cmake/help/latest/command/find_package.html?highlight=find_package

Reply:
Answer below (SDE may be independent from PAPI) is sufficient to negate this proposition.